### ISSUE TITLE
Talkback is not announcing the value information on 'Document properties' dialogue.

### ISSUE THREAD

**Issue by msftedad** [gdh1995/pdf.js#24]
Attach (recommended) or Link to PDF file here:
 https://mozilla.github.io/pdf.js/web/viewer.html.
[edad (2).pdf](https://github.com/gdh1995/pdf.js/files/14600946/edad.2.pdf)

Configuration:
- Operating system and its version: iPhone 13 Version-17.4
- PDF.js version: 1.4   

Steps to reproduce the problem:

1. Open this PDF viewer https://mozilla.github.io/pdf.js/web/viewer.html.
2. Navigate to 'Tools' control at header and activate it.
3. Navigate to 'Document properties' and activate it.
4. Now navigate through the shown properties.
5. Observe the issue.

What is the expected behavior? (add screenshot)
Talkback should announce the value information only, when user navigates to its values using left/right gestures such as 'No'.

What went wrong? (add screenshot)
Talkback is just announcing the label information, not the value information on 'Document properties' dialogue.Talkback announces, 'Fast Web View' (Label), when user navigates to its value 'No' using left/right gestures.

https://github.com/gdh1995/pdf.js/assets/93735775/87ce6d1c-8ba3-4221-bada-696ededfd677

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by msftedad** [comment]
Hi @gdh1995, Hi, Any update on this issue? Please provide the ETA to fix this issue.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
